src/pages/MapPage.css

```diff
@@ -0,0 +1,8 @@
+.mapButton {
+  position: absolute;
+  right: 5%;
+  top: 10%;
```

Comment:
Working with % looks wild and weird on large screens. I would avoid that.
I quite liked the default positioning of the Fabs, that Ionic shipped with.